### PR TITLE
Completed HW 36

### PR DESCRIPTION
**Pull request by RomanMinevich**

### CODE REVIEW

File: src/main/java/multithreading/Race.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package multithreading;
+
+public class Race {
+
+    public static void main(String[] args) {
+        new ThreadExt().start();
+        new RunnableImpl().run();
```
Comment (Sofasmile) [inline src/main/java/multithreading/Race.java]: in this case we run thread (RunnableImpl) in main thread. Do you know smth about that?

File: src/main/java/multithreading/RunnableImpl.java [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package multithreading;
+
+public class RunnableImpl implements Runnable {
+
+    public void run() {
```
Comment (Sofasmile) [inline src/main/java/multithreading/RunnableImpl.java:11]: we should create object of Counter in main method and pass it in constructor for two threads

File: src/main/java/multithreading/ThreadExt.java [hunk 3]
```diff
@@ -0,0 +1,9 @@
+package multithreading;
+
+public class ThreadExt extends Thread {
+
+    @Override
```
Comment (Sofasmile) [inline src/main/java/multithreading/ThreadExt.java:10]: create constructor

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+                        <target>${jdk.version}</target>
+                        <encoding>${project.build.sourceEncoding}</encoding>
+                    </configuration>
+                </plugin>
+            </plugins>
+        </pluginManagement>
+
+    </build>
+
+</project>
```
Comment (Sofasmile) [inline pom.xml]: add empty line

Comment (RomanMinevich) [reply pom.xml]: Added

File: src/main/java/multithreading/ThreadExt.java (hunk 3 above)
Comment (RomanMinevich) [reply src/main/java/multithreading/ThreadExt.java:10]: Done

File: src/main/java/multithreading/RunnableImpl.java (hunk 2 above)
Comment (RomanMinevich) [reply src/main/java/multithreading/RunnableImpl.java:11]: Got it, thanks

File: src/main/java/multithreading/Race.java (hunk 1 above)
Comment (RomanMinevich) [reply src/main/java/multithreading/Race.java]: Replaced "new RunnableImpl().run()" with "new Thread(new RunnableImpl(counter)).start()"

**Review by Sofasmile — APPROVED** [review]

**Pull request merged**